src/components/ItemTable/ItemTableCell.tsx

```diff
@@ -0,0 +1,23 @@
+import { Text } from "@nypl/design-system-react-components"
+
+/**
+ * The ItemTableCell component renders a table cell for the ItemTable
+ *
+ * Its extraction into a separate component allows for styling and other overrides on plain text cells
+ * when rendered programatically in the ItemTableData class
+ */
+const ItemTableCell = ({ children }) => {
```

Comment:
The problem with this is that it's rendering a `p` element inside a `span` which isn't valid HTML5. This might be better as a `Box` rendered as a span.